### ISSUE TITLE
ctrl+v no longer pastes in insert mode

### ISSUE THREAD

**Issue by diminutivesloop** [VSCodeVim/Vim#2646]
Issue Type: <b>Bug</b>

In the past I've been able to paste text via ctrl+v when in insert mode. Recently however that shortcut has stopped working.

Extension version: 0.11.6
VS Code version: Code 1.23.0 (7c7da59c2333a1306c41e6e7b68d7f0caa7b3d45, 2018-05-03T16:44:55.614Z)
OS version: Windows_NT x64 6.1.7601

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4930MX CPU @ 3.00GHz (8 x 3192)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: unavailable_software<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.94GB (18.82GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|yes|
|VM|0%|

</details>

**Comment by januwA** [comment]
该怎么办。。。

**Comment by siehc** [comment]
ctrl + shift + v works for me.

**Comment by januwA** [comment]
hi
Copy: "*yy
Paste: "*p

**Comment by Meli55a** [comment]
`vim.useCtrlKeys:false`, now, you can use `ctrl + c, ctrl + f, ctrl+d`... etc.

**Comment by diminutivesloop** [comment]
Ok but if there is no vim command that conflicts with ctrl+v then. I shouldn't need to set `vim.useCtrlKeys` to `false` right? If so then this still looks like a bug since ctrl+v doesn't appear to be an implemented vim command in this extension.

**Comment by xconverge** [comment]
ctrl+v is for visual block mode

**Comment by ChillingVan** [comment]
And I also try 
`
        {
            "before": [
                "<C-v>"
            ],
            "after": [
                ""
            ]
            "commands": [
            {
                "command": "editor.action.clipboardPasteAction",
                "args": []
            }
        ]
        }
`

But I don't know why it cause my vscode stuck at every input so I cannot use this setting.

**Comment by Echostrike36** [comment]
I enjoyed being able to use Ctrl-v from insert mode to paste. 
</br>
</br>
It worked like this:
</br>
if normal mode > C-v = Visual Block Mode
</br>
if insert mode > C-v = Paste
</br>
</br>
Is this a side effect of #2581?

**Comment by tyru** [comment]
Possibly same as #2624.
This should be fixed at master, or the next release.

**Comment by cheako** [comment]
https://stackoverflow.com/q/50631313/1153319

**Comment by xconverge** [comment]
fixed in #2762
